diagnostic_feedback/templates/underscore/question.html

```diff
@@ -0,0 +1,71 @@
+               name="question[<%- order %>][title]" class="required question-title" required value="<%- title %>">
+        <a href="#" data-tooltip="<%- gettext('Delete Question') %>" class="delete-question delete-button action-button">
+            <i class="icon fa fa-trash-o"></i>
+            <span class="sr">Delete</span>
+        </a>
+
+        <div class="question-txt-editor"><textarea placeholder="<%- gettext('Enter your question') %>"
+                                                   id="question[<%- order %>][text]"
+                                                   name="question[<%- order %>][text]"
+                                                   rows="5" cols="50" class="required question-txt custom-textarea"
+                                                   required><%- text %></textarea></div>
+
+    </div>
+    <h4>Choices</h4>
+    <ol type="A">
+        <% _.each(choices, function(choice) {  %>
+        <li class="answer-choice">
+            <input placeholder="<%- gettext('Choice Text') %>" type="text"
+                   id="question[<%- order %>]answer[<%- choice.c_order %>]"
+                   name="question[<%- order %>]answer[<%- choice.c_order %>]" class="required answer-txt" required
+                   value="<%- choice.name %>"/>
+            <% if(choice.quiz_type == "") { %>
+            <select id="question[<%- order %>]category[<%- choice.c_order %>]"
+                    name="question[<%- order %>]category[<%- choice.c_order %>]" class="required result-choice"
+                    required>
+                <option value=""><%- gettext('Choose result') %></option>
+                <% _.each(choice.resultChoicesOptions, function(result) { %>
+                <option value="<%- result.id %>" <% if(result.id == choice.category_id) { %>
+                        selected="selected" <% } %> ><%- result.name %></option>
+                <% }); %>
+            </select>
+            <input placeholder="<%- gettext('Choice Value') %>" type="number" step="0.1"
+                   id="question[<%- choice.q_order %>]value[<%- choice.c_order %>]"
+                   name="question[<%- order %>]value[<%- choice.c_order %>]" class="required answer-value" required
+                   value="<%- choice.value %>"/>
+
+            <% } else if(choice.quiz_type == choice.BUZZFEED_QUIZ_VALUE){ %>
+            <select id="question[<%- order %>]category[<%- choice.c_order %>]"
+                    name="question[<%- order %>]category[<%- choice.c_order %>]" class="required result-choice"
+                    required>
+                <option value=""><%- gettext('Choose result') %></option>
+                <% _.each(choice.resultChoicesOptions, function(result) { %>
+                <option value="<%- result.id %>" <% if(result.id == choice.category_id) { %>
+                        selected="selected" <% } %> ><%- result.name %></option>
+                <% }); %>
+            </select>
+            <% } else { %>
```

Comment:
It's a little unclear in the code how this `if` statement works at first, and the HTML is repeated. You could avoid repeating this HTML by changing from one `if` statement with three branches to two separate `if` statements:
Change
`if (quiz_type == "") { A, B } else if (quiz_type == BUZZFEED_QUIZ_VALUE) { A } else { B }` 
to
`if (quiz_type == BUZZFEED_QUIZ_VALUE || quiz_type == "") { A }`
`if (quiz_type == DIAGNOSTIC_QUIZ_VALUE || quiz_type == "") { B }`